static/app/utils/fields/index.ts

```diff
@@ -1087,50 +1087,64 @@ export const DISCOVER_FIELDS = [
 ];
 
 enum ReplayFieldKey {
+  ACTIVITY = 'activity',
   BROWSER_NAME = 'browser.name',
   BROWSER_VERSION = 'browser.version',
   COUNT_ERRORS = 'count_errors',
   COUNT_SEGMENTS = 'count_segments',
-  // COUNT_URLS = 'count_urls',
-  DEVICE_MODEL = 'device.model',
+  COUNT_URLS = 'count_urls',
   DURATION = 'duration',
-  // ERROR_IDS = 'error_ids',
-  // LONGEST_TRANSACTION = 'longest_transaction',
+  ERROR_IDS = 'error_ids',
   OS_NAME = 'os.name',
   OS_VERSION = 'os.version',
-  RELEASES = 'releases',
-  // TRACE_IDS = 'trace_ids',
   URLS = 'urls',
-  USER_IP_ADDRESS = 'user.ip',
-  USER_NAME = 'user.username',
 }
 
+/**
+ * Some fields inside the ReplayRecord type are intentionally omitted:
+ * `environment` -> Not backend support, omitted because we have a dropdown for it
+ * `finishedAt` -> No backend support, omitted because we StartDate dropdown and duration field support
+ * `startedAt` -> No backend support, Omitted because we have StartDate dropdown
+ * `longestTransaction` -> value is always zero
+ * `title` -> value is always the empty string
+ */
 export const REPLAY_FIELDS = [
+  ReplayFieldKey.ACTIVITY,
   ReplayFieldKey.BROWSER_NAME,
   ReplayFieldKey.BROWSER_VERSION,
   ReplayFieldKey.COUNT_ERRORS,
   ReplayFieldKey.COUNT_SEGMENTS,
+  ReplayFieldKey.COUNT_URLS,
   FieldKey.DEVICE_BRAND,
   FieldKey.DEVICE_FAMILY,
-  ReplayFieldKey.DEVICE_MODEL,
+  FieldKey.DEVICE_MODEL_ID,
   FieldKey.DEVICE_NAME,
   FieldKey.DIST,
   ReplayFieldKey.DURATION,
+  ReplayFieldKey.ERROR_IDS,
   FieldKey.ID,
   ReplayFieldKey.OS_NAME,
   ReplayFieldKey.OS_VERSION,
   FieldKey.PLATFORM,
-  ReplayFieldKey.RELEASES,
```

Comment:
Does this mean `releases` is no longer a searchable field?

Reply:
`release` and `releases` are both searchable, but the docs will use `release` because more often than not (i assume) the user will start with one value to search by